src/unitTest/PageParserTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+include_once '../parserPages/PageParser.php';
+include_once '../dataBase/XmlFeedModel.php';
+
+use PHPUnit\Framework\TestCase;
+
+class PageParserTest extends TestCase
+{
+
+    protected $objectPage;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->objectPage = new PageParser();
```

Comment:
Why do you need a completely different name of the variable than the one the you assign?
` $this->pageParser = new PageParser();` would be much easier to understand.